### DISCUSSION TITLE
Help with ESP32-Cam and DHT11 needed

### DISCUSSION THREAD

**Discussion by Carl1509** [arendst/Tasmota#18241] [Q&A] [answered]
Hey,

I'm relatively new here and only worked with prebuild binaries.

So I have an AI-Thinker Cam and flashed it with "tasmota32-webcam" binary. 
Now I would like to add a DHT11 sensor to it, but in the "Configure Module" section I can't choose the DHT11.
AFAIK I need to enable the DHT11 capability before compiling, but since I just have the binary I can't dp that.

Is there another way or is there a repository with uncompiled versions of the prebuild?

Thanks :)

**Comment by sfromis** [comment, marked as answer]
The regular source repository includes the setup for the webcam build.

As the webcam function presses the ESP32 hard, inclusion of features and sensors was cut down pretty hard, thus you'd have to replicate the webcam setup found there to add features. This would be against recommendations, but if you want to experiment on your own....

Anyway, the DHT11 cannot be recommended, not only because the timing issues handling this low quality sensor would lead to increased risk of interference with the the CPU load of running webcam.

**Reply by Carl1509** [reply]
Ok thanks

In the repository there are many ways to compile tasmota. Can you recomend one method for beginners?
